packages/wallet-sdk/src/sign/walletlink/WalletLinkSigner.ts

```diff
@@ -492,13 +485,9 @@ export class WalletLinkSigner implements Signer {
     return ensureIntNumber(chainId);
   }
 
-  private async _eth_requestAccounts(): Promise<JSONRPCResponse> {
+  async handshake() {
```

Comment:
this logic below is what we want from WalletLinkSigner for its handshake process.
instead of invoking it indirectly via `return this.request({ method: 'eth_requestAccounts' });`, we can rename this function to be our `handshake` definition.
![image](https://github.com/user-attachments/assets/6db825c1-1b40-4d70-9ef4-7665d1cbf802)